cdap-ui/templates/cdap-etl-batch/S3Avro.json

```diff
@@ -0,0 +1,37 @@
+{
+  "id": "S3Avro",
+  "groups" : {
+    "position": [ "group1" ],
+    "group1": {
+      "display" : "S3 Sink - Avro",
+      "position" : [ "accessID", "accessKey", "basePath", "pathFormat"],
+      "fields" : {
+        "accessID" : {
+          "widget": "textbox",
+          "label": "Access ID"
+        },
+
+        "accessKey" : {
+          "widget": "textbox",
```

Comment:
No people should be able to see the accessKey. It is generated by S3 and not something user selects.